models/intel/smartlab-action-recognition-encoder-0001/accuracy-check.yml

```diff
@@ -14,13 +14,12 @@ evaluations:
       launchers:
         - framework: dlsdk
           # to an appropriate representation with adapter
-          adapter: classification
-          # adapter: action_detection
+          # adapter: classification
 
       datasets:
         - name: DSI1867
           data_source: C:\Users\kminemur\DSI-1867_small\action_recognition\streams_3\high
```

Comment:
we use relative paths to dataset, please remove

Reply:
removed